sources/models/shared-defaults/boot.toml

```diff
@@ -0,0 +1,9 @@
+# Boot related settings
+
+[metadata.settings.boot]
+affected-services = ["bootconfig"]
+setting-generator = "/usr/bin/prairiedog generate-boot-settings"
+
+[services.bootconfig]
+configuration-files = []
+restart-commands = ["/usr/bin/prairiedog generate-boot-config"]
```

Comment:
If either of `settings.boot.kernel-parameters` or `settings.boot.init-parameters` is defined, sundog would skip over settings generation for `settings.boot` to avoid clobbering it by checking if any populated_setting has the target setting as prefix, in this case, `settings.boot`. Since the generation would be for everything under `settings.boot` and prairiedog only returns the whole `settings.boot` serialized json output.